### PR TITLE
v4.1.0-beta

### PR DESCRIPTION
**Pull request by garronej**
Try out https://github.com/garronej/tss-react/pull/105

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
+        },
+        "./nextJs": {
+            "import": "./dist/esm/nextJs.js"
+        },
+        "./compat": {
+            "import": "./dist/esm/compat.js"
+        },
+        "./mui_compat": {
+            "import": "./dist/esm/mui_compat.js"
+        }
```
Comment (garronej) [inline package.json:27]: I ended up removing "require" as I don't see the point of keeping them. Agree @jiby-aurum ?

Comment (jiby-aurum) [reply package.json:27]: @garronej unfortunately I think its required, I tried it 4.1.0 and it fails to import `tss-react/mui` complaining that it is not defined in exports, adding the "require" back fixes them.

Comment (garronej) [reply package.json:27]: @jiby-aurum Thank you for the feedback, I added it back in.  
It's in 4.1.1

Comment (jiby-aurum) [reply package.json:27]: thanks again @garronej just tested, works perfectly !

File: tsconfig.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
     "compilerOptions": {
-        "module": "CommonJS",
```
Comment (jiby-aurum) [inline tsconfig.json:3]: @garronej this should not have been removed,

Comment (garronej) [reply tsconfig.json:3]: Yes, my bad.  
The default config is for CommonJS and we overwrite some values when  building for ESM....

**Pull request merged**